docs/v0.7/en/reference/sql/range.md

```diff
@@ -60,74 +63,135 @@ ALIGN '5s' BY (host) FILL PREV;
 | `LINEAR` | Use [linear interpolation](https://en.wikipedia.org/wiki/Linear_interpolation) to fill the data. If an integer type is filled with `LINEAR`, the variable type of the column will be implicitly converted to a floating point type during calculation |
 |   `X`    |                                           Fill in a constant, the data type of the constant must be consistent with the variable type of the Range expression                                            |
 
+:::tip NOTE
```

Comment:
A note shouldn't be too long, and please avoid using too many notes.
Here seems to be a special case of the `FILL` option, so please remove the `:::tip NOTE` and put it at the bottom of the introduction of the `FILL` option.